cmd/openshift/main.go

```diff
@@ -26,5 +27,6 @@ func main() {
 	sharedmain.Main("tekton-operator",
 		tektonpipeline.NewController,
 		tektontrigger.NewController,
+		rbac.NewController,
```

Comment:
How this `rbac.NewController` will be called :thinking:

Reply:
@savitaashture `sharedmain.Main` does that :wink:

Reply:
My bad i have confused myself with invocation part :grin: